tests/test_buffer_block.py

```diff
@@ -130,3 +130,23 @@ def test_emit_command_groups(self, patched_job):
         self.assert_num_signals_notified(6, block)
         block.stop()
         self.assertFalse(patched_job.call_count)
+
+    def test_buffer_start_signal(self):
+        event = Event()
+        block = EventBuffer(event)
+        block._backup = MagicMock()
+        self.configure_block(block, {
+            "interval": {
+                "milliseconds": 200
+            },
+            "signal_start": True
+        })
+        block.start()
+        self.assert_num_signals_notified(0, block)
+        event.wait(.1)
+        block.process_signals([Signal()])
+        event.wait(.1) # 200 miliseconds have now passed but the block should have only been active for 100
+        block.process_signals([Signal()])
+        event.wait(.1)
+        self.assert_num_signals_notified(2, block) # This would be 1 if signal_start is False
```

Comment:
I would argue that this inline comment and the comment and the one on line 148 should be "block comments", and it would allow you to keep the line length < 79 characters
https://www.python.org/dev/peps/pep-0008/#comments